test/support/module_based_registry.ex

```diff
@@ -13,3 +13,45 @@ defmodule TestRegistry2 do
     {:ok, Keyword.put(options, :members, [:init_test_1, :init_test_2])}
   end
 end
+
+defmodule TestRegistry3 do
+  use Horde.Registry
+  @base_opts [keys: :unique, name: __MODULE__]
```

Comment:
Included so as to demonstrate that valid `child_spec/1` can indeed be generated to facilitate the objective of: `Supervisor.start_link([TestRegistry3], strategy: :one_for_one)`